src/test/utils.tsx

```diff
@@ -218,6 +218,8 @@ export const getFormFields = () => {
     otherInfo: () => screen.getByRole("textbox", { name: /other information/i }),
     submitButton: () =>
       screen.getByRole("button", { name: "Register my account" }),
+    recaptcha: () =>
+    screen.getByTestId("mock-recaptcha-checkbox")
```

Comment:
Small one: the indentation is a bit off. This can be set in one line anyway, I think.